app/script/view_model/list/StartUIViewModel.js

```diff
@@ -151,9 +151,7 @@ z.ViewModel.list.StartUIViewModel = class StartUIViewModel {
 
     this.showNoContacts = ko.pureComputed(() => !this.isTeam() && !this.showContent());
     this.showInviteMember = ko.pureComputed(() => {
-      return (
-        this.selfUser().isTeamOwner() && this.teamSize() === 1 && !this.showContacts() && !this.showSearchResults()
-      );
+      return this.selfUser().isTeamOwner() && this.teamSize() === 1 && !this.showSearchResults();
```

Comment:
Does this really need an explicit return?

Reply:
Yes. IMHO it is much more readable this way if you have to break it up in multiple lines anyway.